include/wx/generic/listctrl.h

```diff
@@ -208,6 +208,8 @@ class WXDLLIMPEXP_CORE wxGenericListCtrl: public wxNavigationEnabled<wxListCtrlB
     wxListHeaderWindow  *m_headerWin;
     wxListMainWindow    *m_mainWin;
 
+    void SetListRulesAlternateColourOnBlank(const bool state, const wxColour& colour);
```

Comment:
```suggestion
    void SetListRulesAlternateColourOnBlank(bool state, const wxColour& colour);
```
wx doesn't use (meaningless) top-level const.